pkg/cmd/pr/view/view.go

```diff
@@ -140,22 +167,46 @@ func printRawPrPreview(io *iostreams.IOStreams, pr *api.PullRequest) error {
 	return nil
 }
 
+func printRawPrComments(out io.Writer, pr *api.PullRequest) error {
+	var b strings.Builder
+	for _, comment := range pr.Comments.Nodes {
+		fmt.Fprint(&b, formatRawPrComment(comment))
+	}
+	fmt.Fprint(out, b.String())
+	return nil
+}
+
+func formatRawPrComment(comment api.Comment) string {
```

Comment:
Do you think that instead of a specialized `formatRawPrComment`, PR comment rendering can reuse the function used by issue comments?